proto/cosmos/tx/v1beta1/tx.proto

```diff
@@ -169,4 +169,10 @@ message Fee {
   // gas_limit is the maximum gas that can be used in transaction processing
   // before an out of gas error occurs
   uint64 gas_limit = 2;
+
+  // if unset, the first signer is responsible for paying the fees. If set, the specified account must pay the fees.
+  // the payer must be a tx signer (and thus have signed this field in AuthInfo).
+  // setting this field does *not* change the ordering of required signers for the transaction.
+  bytes payer = 3
+      [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress", (gogoproto.moretags) = "yaml:\"payer\""];
```

Comment:
Addresses are strings in proto now (#7242).
```suggestion
  string payer = 3 [(gogoproto.moretags) = "yaml:\"payer\""];
```

Reply:
Thanks for the tip. This got merged between when I started and when I finished this pr. You guys are 🚀

Reply:
Issue now resolved. And I rebased on master branch once again to keep this up to date